src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -2155,27 +2156,27 @@ GeneratorController::GeneratorController(EnergyPlusData &state,
     if (UtilityRoutines::SameString(objectType, "Generator:InternalCombustionEngine")) {
```

Comment:
Use `constexpr std::array<std::string_view>` and `getEnumerationValue`.

Reply:
I made the changes in [this commit](https://github.com/NREL/EnergyPlus/pull/9053/commits/b82197eb8e5a5b8f1f193cad5abf8f2e4d52fb7e). Does this look good? 

PS: Ignore the `std::array` name, I changed it in the next commit.

Reply:
See comments on the individual commits. In this particular case you missed an opportunity which I pointed out.